packages/contracts-bedrock/contracts/deployment/MigrationSystemDictator.sol

```diff
@@ -229,4 +252,23 @@ contract MigrationSystemDictator is BaseSystemDictator {
 
         finalized = true;
     }
+
+    /**
+     * @notice First exit point, can only be called before step 3 is executed.
```

Comment:
I don't think before is accurate here because of the require statement that checks the `currentStep == EXIT_1_NO_RETURN_STEP` - it must be called at an exact place

Reply:
Should it be `<=` instead of `==`?

Reply:
It must be `==` because `DTL_SHUTOFF_BLOCK` and `oldL1CrossDomainMessenger` are not set until `EXIT_1_NO_RETURN_STEP`.